lib/chip-types/collection.js

```diff
@@ -72,7 +74,7 @@ Collection.prototype.get_unknown_field_errors = function(values){
 	const validation_errors = {};
 	for (const field_name in values){
 		if (this.fields[field_name] === undefined){
-			validation_errors[field_name] = new Errors.ValidationError(`Unknown field '${field_name}' for resource-type '${this.name}`);
+			validation_errors[field_name] = new Boom.badRequest(`Unknown field '${field_name}' for collection '${this.name}`);
```

Comment:
afaik, `badRequest` is for malformet requests (like json syntax problems). I think that `forbidden` is more appropriate here. (see [reference](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.4))